co-log/src/Colog/Actions.hs

```diff
@@ -26,10 +26,12 @@ module Colog.Actions
        ) where
 
 import Control.Monad.IO.Class (MonadIO (..))
+import Data.Semigroup ((<>))
```

Comment:
This import is needed on old GHC versions and produces warnings on new versions. Since there already are some warnings about unused `Data.Semigroup` I concluded that I shouldn't do anything about that, but I can add CPP macros to solve it.